pkg/netpol/eval/service_test.go

```diff
@@ -0,0 +1,64 @@
+package eval
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/np-guard/netpol-analyzer/pkg/netpol/internal/testutils"
+)
+
+type serviceMapping struct {
+	serviceName      string
+	serviceNamespace string
+	numPods          int
+}
+
+func TestServiceMappingToPods(t *testing.T) {
```

Comment:
add cases with a service that does not exist (with cases for existing and non existing namespace), and a case for a service that has no pods.